guide/commando/README.md

```diff
@@ -0,0 +1,142 @@
+# Getting Started with Commando
```

Comment:
Top-level headers should always use `##` instead of `#`, because Docsify won't render them otherwise. It's useful to have them on the sidebar for when you want to scroll to a certain part of the page easily.

Do note that this basically means you just need to add one `#` to all headers on the page(s).